README.md

```diff
@@ -1,23 +1,92 @@
-# Developers Italia website (https://developers.italia.it)
-
-[![Netlify Status](https://api.netlify.com/api/v1/badges/92a97b26-4e6c-4408-9270-9603f951eccf/deploy-status)](https://app.netlify.com/sites/developers-italia/deploys) [![CircleCI](https://circleci.com/gh/italia/developers.italia.it.svg?style=svg)](https://circleci.com/gh/italia/developers.italia.it)
-
-The website of the developer community designing and developing public digital services in Italy.
-
-The website is developed using [Jekyll](https://jekyllrb.com/) and it's currently served through [GitHub pages](https://pages.github.com/).
-
-A [CircleCI job](.circleci/config.yml) builds the sources and commits the resulting artifacts to GitHub, in the [gh-pages branch](https://github.com/italia/developers.italia.it/tree/gh-pages). The [same job](.circleci/config.yml) is also triggered every night to get the most updated data feeding the website.
-
+<h1 align="center">Developers Italia</h1>
+
+<div align="center">
+<img src=".github/logo.png" width="200">
+<br />
+<i> We develop public services, together </i>
+</div>
+
+<br />
+
+ <!-- Badges -->
+<div align="center">
+  <!-- CircleCI-->
+  <a href="https://circleci.com/gh/italia/developers.italia.it">
+    <img alt="CircleCI" src="https://circleci.com/gh/italia/developers.italia.it.svg?style=shield">
+  </a>
+  <a href="https://app.netlify.com/sites/developers-italia/deploys">
+    <img alt="Netlify" src="https://img.shields.io/netlify/92a97b26-4e6c-4408-9270-9603f951eccf">
+  </a>
+  <a href="https://github.com/italia/developers.italia.it/releases">
+    <img alt="GitHub tag (latest by date)" src="https://img.shields.io/github/v/tag/italia/developers.italia.it">
+  </a>
+  <a href="LICENSE.md">
+    <img alt="LICENSE" src="https://img.shields.io/github/license/italia/developers.italia.it">
+  </a>
+</div>
+
+<div align="center">
+  <h3>
+    <a href="https://developers.italia.it">
+      Website
+    </a>
+    <span> | </span>
+    <a href="https://developers.italia.it/en/software">
+      Reuse Catalog
+    </a>
+    <span> | </span>
+    <a href="https://github.com/italia/publiccode.yml">
+      publiccode.yml
+    </a>
+    <span> | </span>
+    <a href="CONTRIBUTING.md">
+      Contributing
+    </a>
+  </h3>
+</div>
+
+# Index
+
+- [Description](#description)
+- [Documentation](#documentation)
+- [Community](#community)
+- [License](#license)
+
+# Description
+
+<div align="center">
+  <img alt="Developers Italia Homepage" src=".github/screenshot.png" />
```

Comment:
I'm not sure about the screenshot, it takes away a lot of focus and it'll be outdated pretty quickly by news and other changes.

After all the website is just one click away.

Also, with no screenshot to push the content down we could get away with having no Index, for now.

Reply:
Ok makes sense, let's get rid of those things for now.